nightguard WatchKit Extension/InterfaceController.swift

```diff
@@ -244,19 +261,19 @@ class InterfaceController: WKInterfaceController, WKCrownDelegate {
     
     fileprivate func assureThatBaseUriIsExisting() {
         
-        if UserDefaultsRepository.readBaseUri().isEmpty {
+        if UserDefaultsRepository.baseUri.value.isEmpty {
             AppMessageService.singleton.requestBaseUri()
         }
     }
     
     fileprivate func assureThatDisplayUnitsIsDefined() {
         
-        if !UserDefaultsRepository.areUnitsDefined() {
-            // try to determine whether the user wishes to see value in mmol or mg/dL
-            NightscoutService.singleton.readStatus { (units) in
-                UserDefaultsRepository.saveUnits(units)
-            }
-        }
+//        if !UserDefaultsRepository.units.exists {
```

Comment:
We don't need it anymore. On watch activation a WatchSyncRequestMessage is sent and the phone will send back a UserDefaultSyncMessage if the watch didn't have the same settings, updating all the shared settings with phone values (URI, display units, alarm settings, etc).

In more detail: for each change on any of the phone settings that are shared with the watch (the "watchSync" group), an "sync update ID" is generated and stored on phone; also, a UserDefaultSyncMessage message with all the phone settings that are shared with the watch, plus this ID is sent to watch. 
Because this message is not always received by watch (God knows why...), on each watch activation a UserDefaultSyncMessage is sent to phone to verify that these two "last sync update IDs" are the same. 
That is why the "assure..." methods are not needed anymore. But, I noticed there is a displaying glitch when the URI is empty on the watch (on a fresh install), I'll fix it next.

Reply:
Here is the fix I talked about: https://github.com/florianpreknya/nightguard/commit/205afcad5de5a1b19667ca726ecd9cc5a3226e25

Do you want to make a new PR with the fix?

Reply:
Great. As you like.

Reply:
I can simply pull your commit if that's easier...

Reply:
I checked, the fixes are already on this PR.